github/orgs_rules_test.go

```diff
@@ -18,14 +18,16 @@ func TestOrganizationsService_GetAllOrganizationRulesets(t *testing.T) {
 	t.Parallel()
 	client, mux, _ := setup(t)
 
-	mux.HandleFunc("/orgs/o/rulesets", func(w http.ResponseWriter, r *http.Request) {
+	org := "o"
+
+	mux.HandleFunc(fmt.Sprintf("/orgs/%s/rulesets", org), func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		fmt.Fprint(w, `[{
+		fmt.Fprintf(w, `[{
 			"id": 26110,
 			"name": "test ruleset",
 			"target": "branch",
 			"source_type": "Organization",
-			"source": "o",
+			"source": "%s",
```

Comment:
Definitely, please revert.  Placing this `"%s"` so far away from its value is really not helpful.
Additionally, we attempt to use `"%v"` throughout this repo unless we want to call specific attention as to why it important _not_ to be `"%v"`.

Reply:
Sure.